mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -501,7 +501,7 @@ def test_read_vhdr_annotations_and_events():
     raw.annotations.append([1, 2, 3], 10, ['ZZZ', s_10, 'YYY'])
```

Comment:
The version from e64fd4fb7 simplifies `_check_event_id` and allow to remove `getattr` and the class methods. But requires to pass `event_id` because we are appending extra markers.

```diff
-    _, event_id = events_from_annotations(raw)
+    _, event_id = events_from_annotations(raw, event_id='brainvision')
```

If we want to keep the default, in this case, we need to keep the class method etc. Or relax
`_check_bv_annot` and as soon as we detect one 'brainvision' marker, parse everything as 'brainvision'

WDYT @agramfort @jona-sassenhagen? How would you like to write the call?